build-system/tasks/karma.conf.js

```diff
@@ -302,13 +302,13 @@ module.exports = {
   captureTimeout: 4 * 60 * 1000,
   failOnEmptyTestSuite: false,
 
-  // AMP tests on Sauce take ~9 minutes, so don't fail if the browser doesn't
-  // communicate with the proxy for up to 10 minutes.
+  // AMP tests on Sauce can take upto 12-13 minutes, so don't fail if the
```

Comment:
nit: `up to`

Reply:
Oops, didn't see this on mobile UI. Will fix next time.